### PR TITLE
better 2 profiles guide

### PR DESCRIPTION
**Pull request by lasote**

### CODE REVIEW

File: conan_v2.rst [hunk 1]
```diff
@@ -71,9 +71,14 @@ available for Conan 2.0.
 Host and build profiles and new cross-building model
 ----------------------------------------------------
 
-Use always build and host profiles. You can enable it by passing ``-pr:b=default`` in the
-command line to most commands. Do not use ``os_build``, ``arch_build`` anywhere in your
-recipes or code.
+Use always build and host profiles.
```
Comment (czoido) [inline conan_v2.rst:74]: maybe we can add a link to: https://docs.conan.io/en/latest/reference/profiles.html#build-profiles-and-host-profiles ?

Comment (lasote) [reply conan_v2.rst:74]: sure!

**Review by memsharded — APPROVED** [review]

File: conan_v2.rst (hunk 1 above)
Comment (lasote) [reply conan_v2.rst:74]: oh, you meant in that paragraph, wait 1 min

File: conan_v2.rst [hunk 2]
```diff
@@ -71,9 +71,14 @@ available for Conan 2.0.
...
+You can enable it:
```
Comment (czoido) [inline conan_v2.rst:75]: Maybe we can add extend the sentence a bit? Like: _Conan 1.x will use one profile by default, to start using two profiles, please do the following_ or something like that?

Comment (lasote) [reply conan_v2.rst:75]: sure

**Review by czoido — APPROVED** [review]

**Pull request merged**